### ISSUE TITLE
Keyboard Shortcut incorrect on British keyboard

### ISSUE THREAD

**Issue by ShayHurley** [Microsoft/vscode#25647]
- VSCode Version: 1.11.2
- OS Version: 10.11.6
- MacBook Pro using built-in British keyboard layout - no external keyboard.

Steps to Reproduce:

**Note**: You'll need a mac with a British keyboard layout for the below scenario 

1. Open the Keyboard Shortcut Panel  `⌘K ⌘S`
2. Search for Terminal in the list of shortcuts
3. The keyboard shortcut displayed is ``⌃` ``

**Note**: this shortcut combination will not open the terminal.

4. Hover on the terminal command in the keyboards shortcut list and click the pen symbol to the left to edit the entry.
5. On a British keyboard press the key combination `` ⌃` `` to set the shortcut combination

**Note**: The symbols displayed in the popup are ``⌃§`` ( ctrl+[IntlBackslash] ) which is on a different key on a British keyboard. The IntlBackslash key is located to the left of the number one on the main keyboard.

**Comment by alexandrudima** [comment]
Duplicate of #24153

TL;DR `[Backquote]` and `[IntlBackslash]` were swapped on ISO keyboards due to Chromium swapping them without our knowledge. We've added a band-aid where we now try to detect ISO keyboards and try to unswap the two scan codes.
